src/searchbar/Searchbar.stories.tsx

```diff
@@ -166,3 +169,39 @@ fullSearchbar.parameters = {
         disable: true,
     },
 };
+
+export const searchbarWithDateChip = () => <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
The change value for 'date only' pickers also include a timestamp.
This is the ts of the user clicked on the datepicker.

![image](https://user-images.githubusercontent.com/34885540/112822847-c0d3f380-9088-11eb-9a29-1584240337f9.png)


Shouldn't we hardcode the hour to be 00:00 ? or does the hour get ignored when creating a search query in finder with 'date only' pickers?

Reply:
Good observation.

However, actually handing the data that is sent in the events is the job of the application that uses this UI component.
The UI component itself does not know the difference between date and datetime properties.
But we will need to check that it works as expected in the application.